README.md

```diff
@@ -24,6 +24,7 @@ For information about the Kogito Operator architecture and instructions for usin
       * [Running performance tests](#running-performance-tests)
       * [List of test tags](#list-of-test-tags)
     * [Running the Kogito Operator locally](#running-the-kogito-operator-locally)
+    * [Remote Debug Kogito Operator using Intellij IDEA](#remote-debug-kogito-operator)
```

Comment:
```suggestion
    * [Remote Debug Kogito Operator using Intellij IDEA](#remote-debug-kogito-operator-using-intellij-idea)
```
Same as heading for the link to work :)

Reply:
Thanks for pointed it out. I had fixed it now